botocore/httpchecksum.py

```diff
@@ -432,6 +435,16 @@ def _apply_request_trailer_checksum(request):
     )
 
 
+def _register_checksum_algorithm_feature_id(algorithm):
+    checksum_algorithm_name = algorithm["algorithm"]
+    if checksum_algorithm_name.upper() == "CRC64NVME":
+        checksum_algorithm_name = "CRC64"
+    checksum_algorithm_name_feature_id = (
+        "FLEXIBLE_CHECKSUMS_REQ_" + checksum_algorithm_name.upper()
+    )
```

Comment:
I think it's better practice to use a f-string for cases like this. Something like:
```python
f"FLEXIBLE_CHECKSUMS_REQ_{checksum_algorithm_name}"
```

Note: This assumes you accepted my change above related to `upper()`